docs/sigs/Certification.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Certification SIG"
+---
+# name of SIG
+
+Create and maintain standards and procedures related to hardware and software certification for AlmaLinux.
+
+## How to Join
+
+Email a SIG member or show up on the chat. Occasionally there non-disclosure agreements with hardware vendors, so some chats/meeting might be private, but SIG members will do their best to welcome everyone as fully as possible.
+
+**Where we chat**
+
+We use [mattermost SIG/Certification](https://chat.almalinux.org/almalinux/channels/sigcertification) channel for communication and all are welcome there.
+
+**Where and when we meet**
+
+Meetings are approximately every fortnight. If there's nothing confidential in the next meeting agenda a SIG member will happily welcome you along and provide time and meeting URL information.
+
+## Activities, projects, and deliverables
+
+Current deliverables are:
+* [AlmaLinux OS Foundation Hardware Certification Program](https://docs.google.com/document/d/1r33jRSavBa9yiI8FlRDHbpOhhW83xMFrsR_bqTnSJPI/edit?usp=sharing) - the document describing the process for certifying hardware.
+* The [AlmaLinux Hardware Certification Suite](https://github.com/AlmaLinux/Hardware-Certification-Suite), where the Ansible rules for exercising a test suite is maintained.
+
+Once active, the SIG will be taking test results and issuing certifications.
+
+### Help wanted
+
+* Review of the AlmaLinux OS Foundation Hardware Certification Program document to ensure its clear and concise.
+* Contributions towards expanding the AlmaLinux Hardware Certification Suite to represent a meaningful range of hardware and Linux hardware driver tests that an end user would expect a certification to cover.
+* There is work in progress happening where an Independent Software Vendor can maintain a set of tests for their software with obligations as an addition part of this suite. Once this is defined more, vendor participation will be encouraged. Let us know if this is you so you can be advised when we are ready.
+* Eventually users will be welcome to run the AlmaLinux Hardware Certification Suite on the hardware they have access to and submit these results.
+
+## SIG members
+
+* [Jonathan Wright](mailto:jonathan@almalinux.org) - Committee lead
+* [Akitsu Izawa](mailto:akitsu.izawa@cybertrust.co.jp) - member
+* [Asaka Matsumoto](mailto:asaka.matsumoto@cybertrust.co.jp) - member
```

Comment:
Asaka is presently on leave of absence. I am the temporary successor of her. 
Could you add myself instead? 
- Koichiro Iwao <meta@almalinux.org>

Reply:
done!

Reply:
Asaka Matsumoto is on leave.